options.go

```diff
@@ -94,10 +97,9 @@ func WithUpdatePeriod(updatePeriod time.Duration) Option {
 
 type withResendTimeout struct{ resendTimeout time.Duration }
 
-func (o withResendTimeout) applyProtocol(p *Protocol) { p.resendTimeout = o.resendTimeout }
-func (o withResendTimeout) applyEndpoint(e *Endpoint) { e.resendTimeout = o.resendTimeout }
+func (o withResendTimeout) applyConn(c *Conn) { c.resendTimeout = o.resendTimeout }
```

Comment:
warning: exported function WithResendTimeout should have comment or be unexported (golint)